packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -34235,17 +34230,6 @@ Map {
   },
   "PasswordInput" => Object {
     "$$typeof": Symbol(react.forward_ref),
-    "defaultProps": Object {
```

Comment:
do we need to provide any of these deleted default props? the className for instance.

Reply:
The defaults are still there. Carbon has taken to inlining them instead of using `defaultProps`, so they're not showing up in our snapshots, but as for the className in particular. It was actually broken for a _very_ long time. Notice it's actually showing `${prefix}` instead of `bx` the default wasn't using template strings, but a normal string. https://github.com/carbon-design-system/carbon/blame/c2a62f9ce71c9570e9de21d266ccfe81f0ae3e18/packages/react/src/components/TextInput/PasswordInput.js#L339